changelogs/fragments/74852-apt-allow-downgrade.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - apt - added an ``allow_downgrade`` option to enable safe downgrade of packages without using ``force`` which doesn't verify signatures.
```

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment
Changelog fragment should have a link to the issue or PR, in this case it's best to mention both IMHO.
```suggestion
  - apt - added an ``allow_downgrade`` option to enable safe downgrade of packages without using ``force`` which doesn't verify signatures (https://github.com/ansible/ansible/issues/29451, https://github.com/ansible/ansible/pull/74852).
```